<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDisc</class>
 <widget class="QDialog" name="AddDisc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="radioDisc">
       <property name="minimumSize">
        <size>
         <width>189</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>189</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить дисциплину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioCycle">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>137</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить цикл</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Название дисциплины</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>38</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>38</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Цикл</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Заключение</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="butCan">
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butAdd">
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
